packages/waku/src/lib/renderers/html-renderer.ts

```diff
@@ -44,6 +44,7 @@ Promise.resolve(new Response(new ReadableStream({
 
 const injectRscPayload = (
   readable: ReadableStream,
+  config: ResolvedConfig,
```

Comment:
Let's change this to:
```suggestion
  mainJsPath: string,
```